nextflow.config

```diff
@@ -31,7 +31,16 @@ params {
   fastp = false               // Use fastp trimming tool (short reads) (true, false)
   trim_galore = false         // Use trim_galore trimmer (best for nextera short reads) (true, false)
   
-
+  // Select which optional stages should be run
+  stage_reporting = true      // run this stage, [true, false]
+  stage_haybaler = true       // run this stage, [true, false]
```

Comment:
Some stages require other stages  (for example requires haybaler reporting), we should put that information in here.